ouroboros-consensus/docs/report/chapters/future/genesis.tex

```diff
@@ -60,7 +60,8 @@ \subsection{Background: understanding the Longest Chain rule}
 \end{center}
 %
 This motivates the first part of the Longest Chain rule: chain length is a
-useful proxy for chain density. The second part of the rule---the intersection
+useful proxy for chain density, which in turn reflects honesty.
```

Comment:
Stylistic nitpick: wrap (`M-q`) at 80 chars (same for other changes).

Reply:
I added a final commit that just rewraps the paragraphs I touched in the "real" commits. 👍